### PR TITLE
Fix minor text issues in compile.md

### PR DESCRIPTION
**Pull request by murlakatamenka**
Everything is tested to work.

Environment used:

- Windows 10 x64 (PowerShell turned on)
- cmake 3.10.2
- Visual Studio 2017 Community Edition 15.6
- AMD APP SDK 3.0
- CUDA 9.1 (Patch 3)

### CODE REVIEW

**Comment by fireice-uk** [comment]
Hi. Did you intend fac8c to be in this PR? That's a build script.

**Comment by murlakatamenka** [comment]
Hi, I intended to but at a later time. Sorry for a little confusion.
Anyway I'm finished with what I wanted to PR as for now. Will edit PR title so that it corresponds to its actual context.

File: scripts/build_xmr-stak_windows/build_xmr-stak_all_backends.bat [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+:: NVIDIA GPU
+cmake .. -DCUDA_ENABLE=ON
+:: Lists with CUDA compute architectures: https://developer.nvidia.com/cuda-gpus and https://en.wikipedia.org/wiki/CUDA#GPUs_supported
+:: Quick reference
+:: Architecture: GPUs - corresponding values
+:: Kepler: 	GTX 6xx, 7xx 	- 30;35;37
+:: Maxwell: GTX 9xx			- 50;52
+:: Pascal: 	GTX 10xx		- 60;61;62
+:: Volta: 	GTX Titan V		- 70
+cmake .. -DCUDA_ARCH=30;35;37;50;52;60;61;62;70
```
Comment (psychocrypt) [inline scripts/build_xmr-stak_windows/build_xmr-stak_all_backends.bat]: This work only with cuda 9+. If you leave it empty than all possible versions depending on the install CUDA verison will be created.

File: scripts/build_xmr-stak_windows/clean_build_xmr-stak.bat [hunk 2]
```diff
@@ -0,0 +1,2 @@
+if exist ..\..\build (rd /s /q ..\..\build)
+pause
```
Comment (psychocrypt) [inline scripts/build_xmr-stak_windows/clean_build_xmr-stak.bat]: missing empty line

File: scripts/build_xmr-stak_windows/build_xmr-stak_gpu_nvidia.bat [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+cmake --build . --config Release --target install
+
+echo Build finished: & powershell get-date -format "{dd-MMM-yyyy HH:mm:ss}"
+
+copy ..\xmr-stak-dep\openssl\bin\*.dll .\bin\Release
+cd bin\Release & ren xmr-stak.exe xmr-stak-gpu-nvidia.exe & start .
+
+echo.
+pause
```
Comment (psychocrypt) [inline scripts/build_xmr-stak_windows/build_xmr-stak_gpu_nvidia.bat]: missing empty line

File: scripts/build_xmr-stak_windows/build_xmr-stak_gpu_amd.bat [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+cmake --build . --config Release --target install
+
+echo Build finished: & powershell get-date -format "{dd-MMM-yyyy HH:mm:ss}"
+
+copy ..\xmr-stak-dep\openssl\bin\*.dll .\bin\Release
+cd bin\Release & ren xmr-stak.exe xmr-stak-gpu-amd.exe & start .
+
+echo.
+pause
```
Comment (psychocrypt) [inline scripts/build_xmr-stak_windows/build_xmr-stak_gpu_amd.bat]: missing empty line

File: scripts/build_xmr-stak_windows/build_xmr-stak_cpu.bat [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+cmake --build . --config Release --target install
+
+echo Build finished: & powershell get-date -format "{dd-MMM-yyyy HH:mm:ss}"
+
+copy ..\xmr-stak-dep\openssl\bin\*.dll .\bin\Release
+cd bin\Release & ren xmr-stak.exe xmr-stak-cpu.exe & start .
+
+echo.
+pause
```
Comment (psychocrypt) [inline scripts/build_xmr-stak_windows/build_xmr-stak_cpu.bat]: missing empty line

File: scripts/build_xmr-stak_windows/build_xmr-stak_all_backends.bat [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+cmake --build . --config Release --target install
+
+echo Build finished: & powershell get-date -format "{dd-MMM-yyyy HH:mm:ss}"
+
+copy ..\xmr-stak-dep\openssl\bin\*.dll .\bin\Release
+
+start bin\Release
+
+echo.
+pause
```
Comment (psychocrypt) [inline scripts/build_xmr-stak_windows/build_xmr-stak_all_backends.bat]: missing empty line

**Comment by psychocrypt** [comment]
Thanks for the PR (I annotated a view parts).
I am not sure if we should merge this PR. 
Everything within this hard coded scripts can be derived from the documentation. I am not sure if this only increase our parts we need to check each time if we change the process or an option to build xmr-stak.

**Comment by fireice-uk** [comment]
I'm not against the idea per-se, but the build scripts need to work for everyone (CUDA issue).

**Comment by murlakatamenka** [comment]
Thanks for the review, @psychocrypt , I've fixed the issues in 484f69e.

>  Everything within this hard coded scripts can be derived from the documentation. I am not sure if this only increase our parts we need to check each time if we change the process or an option to build xmr-stak.

That's true, the scripts are obviously heavily based on the documentation ([compile.md](https://github.com/fireice-uk/xmr-stak/doc/compile.md) and [compile_Windows.md](https://github.com/fireice-uk/xmr-stak/compile_Windows.md)) and are rather straightforward (highly verbose, no branching) but at the same time this "dumbness" helps to avoid errors and they just do the job provided the dependencies are met.  
Of course the final decision is yours but feel free to assign me to related issues if case scripts get merged.  

@fireice-uk thanks, as mentioned above it's fixed in 484f69e. Default build option is used for `-DCUDA_ARCH` now, people are expected to provide a value or semicolon separated list otherwise.

**Comment by psychocrypt** [comment]
The description is added with #1266. I am not sure if the scripts are necessary

**Comment by murlakatamenka** [comment]
I'm leaving scripts on my fork (why not after all) and closing this PR.

**Pull request closed without merging**